.github/chatops_commands.md

```diff
@@ -4,3 +4,4 @@
 - :white_check_mark: `/gen-test` - generate test codes
 - :label: `/label` - add labels
 - :rewind: `/rebase` - rebase master
+- :end: :two: :end: `/label actions/e2e-deploy` - run E2E deploy & integration test
```

Comment:
seems good, but how about add another option like /e2e which adds actions/e2e-deploy labels, then executes e2e test by event.

Reply:
thanks, but adding new chatops command takes too much cost. (e.g. maintain, CI processing speed)
so i don't wanna prepare such a thing.

Reply:
I understand just suggestion.
please don't mind

Reply:
sorry for inconvenience :sob: